test/Command.js

```diff
@@ -187,7 +184,70 @@ describe("Command", () => {
   });
 
   describe(".runPreparations()", () => {
-    it("needs tests");
+    describe(".filteredPackages", () => {
+      let testDir;
+
+      beforeEach(() => initFixture("UpdatedCommand/basic").then((dir) => {
+        testDir = dir;
+      }));
+
+      class TestCommand extends Command {}
+
+      function cli(cmd, ...args) {
```

Comment:
Ideally, this would be the approach used for all unit tests of commands. The `.run()` method really needs to return a Promise, already... (out of scope for this PR, clearly)